### DISCUSSION TITLE
"PSF profiler report generator" window doesn't scroll

### DISCUSSION THREAD

**Discussion by JSDLacoste** [MontpellierRessourcesImagerie/MetroloJ_QC#6] [Q&A]
Bonjour de Montréal!  Félicitations pour ce nouvel outil.  Utilisatrice de MetroloJ, heureuse de maintenant essayer MetroloJ_QC.  I do have an issue.  When attempting to generate fwhm, the window "PSF profiler report generator" is longer than my screen and I am unable to scroll up and down.  Hence I am unable to set the parameters and launch the analysis.  Merci pour vos suggestions.

Judith  

My computer:
Hardware is MacBook Pro running  as Ubuntu 18.04.5 LTS
Fiji/ImageJ 1.53k
screen is set to highest possible resolution (1280x800)

**Comment by julien-cau** [comment]
Hi Judith,

Sorry for the late answer. I'll try to fix this, although I am not sure the generic dialog plugin used allows this.
Best

**Reply by JSDLacoste** [reply]
Hi Julien,
Thanks, well appreciated.  Please let me know if I can do something to make it easier for you.
Best regards,
Judith

**Reply by julien-cau** [reply]
Hi Judith,

I am sorry to get back to you only know. I implemented some scrollbar options in version 1.2.0 That was quite straightforward but I first used a plugin that was deprecated. And the last quarter was really busy for me. When I understood if someone would use ImageJ2 this enhancedGenericDialog plugin was not supported I was under some administrative avalanche for months...
 
If still interested, please email me at julien.cau [at] biocampus.cnrs.fr so that I can send you some preliminary version of 1.2.0
Best
Julien
